.bazelrc

```diff
@@ -1,7 +1,7 @@
 # what is defined in this section will be applied at all times
-build --workspace_status_command bin/workspace_status.sh --show_result=0 --show_progress=no --incompatible_strict_action_env
-test --show_result=0 --show_progress=no --incompatible_strict_action_env
-run --show_result=0 --show_progress=no --incompatible_strict_action_env
+build --workspace_status_command bin/workspace_status.sh --incompatible_strict_action_env
+test --show_result=0 --incompatible_strict_action_env
+run --show_result=0 --incompatible_strict_action_env
```

Comment:
What are these changes for, @vmax ?

Reply:
@haikalpribadi it's hard to debug stuff when progress is not shown; Core is the only repo that had these flags set so I think it's appropriate to remove them: `--show_result=0`, `--show_progress=no`

Reply:
right, makes sense. Should we do this across all repos, @vmax ?